<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>okPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="buttonMenu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonStop">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/tango/images/icons/tango/media-playback-stop.png</normaloff>:/tango/images/icons/tango/media-playback-stop.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPause">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pause</string>
        </property>
        <property name="text">
         <string>||</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/tango/images/icons/tango/media-playback-pause.png</normaloff>:/tango/images/icons/tango/media-playback-pause.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPlay">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/tango/images/icons/tango/media-playback-start.png</normaloff>:/tango/images/icons/tango/media-playback-start.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPrev">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Previous</string>
        </property>
        <property name="text">
         <string>|&lt;</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/tango/images/icons/tango/media-skip-backward.png</normaloff>:/tango/images/icons/tango/media-skip-backward.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonNext">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Next</string>
        </property>
        <property name="text">
         <string>&gt;|</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/tango/images/icons/tango/media-skip-forward.png</normaloff>:/tango/images/icons/tango/media-skip-forward.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonRandom">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Random</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/tango/images/icons/tango/dialog-information.png</normaloff>:/tango/images/icons/tango/dialog-information.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderVolume">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="okLotsOfStatesButton" name="buttonPlayingModeState">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="okTrackTimeLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="okSeekSlider" name="sliderSeek">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonSavePlaylist">
        <property name="text">
         <string>Save playlist</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLoadFavorites">
        <property name="text">
         <string>Load favourites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fieldQuickSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
      <item>
       <widget class="okPlaylistListWidget" name="playlistListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="okPlaylistWidget" name="playlistWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="okDirTreeView" name="dirTree">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>okDirTreeView</class>
   <extends>QTreeView</extends>
   <header>okWidgets/okdirtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>okSeekSlider</class>
   <extends>QSlider</extends>
   <header location="global">okWidgets/okseekslider.h</header>
  </customwidget>
  <customwidget>
   <class>okTrackTimeLabel</class>
   <extends>QLabel</extends>
   <header location="global">okWidgets/oktracktimelabel.h</header>
  </customwidget>
  <customwidget>
   <class>okPlaylistWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">okWidgets/okplaylistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>okLotsOfStatesButton</class>
   <extends>QPushButton</extends>
   <header>okWidgets/oklotsofstatesbutton.h</header>
  </customwidget>
  <customwidget>
   <class>okPlaylistListWidget</class>
   <extends>QListWidget</extends>
   <header>okWidgets/okplaylistlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
